data/lua/wml-tags.lua

```diff
@@ -125,7 +125,7 @@ end
 
 function wml_actions.store_unit_type_ids(cfg)
 	local types = {}
-	for k,v in pairs(wesnoth.unit_types) do
+	for k, _ in pairs(wesnoth.unit_types) do
```

Comment:
Remove the unnecessary assignment to `_`:

```lua
for k in pairs(wesnoth.unit_types) do
```

Reply:
IDK, I prefer to write 2 args to remind the reader that there are two underlying variables. But no problem, will fix.

Reply:
I too prefer to write two arguments, actually. Not a fan of using `_` for unused ones though...